lib/Accessibility.js

```diff
@@ -329,7 +329,9 @@ class AXNode {
       role: this._role
     };
 
-    /** @type {!Array<keyof SerializedAXNode>} */
+    /** @enum {'name'|'value'|'description'|'keyshortcuts'|'roledescription'|'valuetext'} */
+    let UserStringProperties; // eslint-disable-line no-unused-vars
```

Comment:
Is there a way to do this without the need for the `let` and the `eslint` comment? (I'm no TypeScript expert...)

Reply:
I am not sure if this actually works, but it is worth a try:

```js
/** @enum {string} */
const UserStringProperties = {
  Name: 'name',
  Value: 'value',
  ...
}
```

Then you can do `Object.values(UserStringProperties)` instead of the `userStringProperties` array down below.

Reply:
I tried this but it unfortunately doesn't work, because when you do `for foo in Object.values(UserStringProperties)` TypeScript only knows that `foo` is a `string`; it loses the fact that it was actually one of `UserStringProperties`, which causes problems. So I think this is the nicest way for now :(